### PR TITLE
Use built in bliss logo directly

### PR DESCRIPTION
**Pull request by utzcoz**

### CODE REVIEW

File: app/src/main/java/com/farmerbb/taskbar/ui/TaskbarController.java [hunk 1]
```diff
@@ -457,15 +456,7 @@ void drawStartButton(Context context, ImageView startButton, SharedPreferences p
                 Drawable drawable;
 
                 if(U.isAndroidGeneric(context)) {
-                    try {
```
Comment (farmerbb) [inline app/src/main/java/com/farmerbb/taskbar/ui/TaskbarController.java:460]: IIRC this was done so that on Android-Generic builds the start button could be themed appropriately for each variant (I think via an RRO?).  This would revert everything back to the previous behavior of showing the Bliss logo on every Android-Generic variant.

@electrikjesus is this okay?  I guess I don't really have any context on why this change is needed.

Comment (electrikjesus) [reply app/src/main/java/com/farmerbb/taskbar/ui/TaskbarController.java:460]: Yeah, We will use RRO overlays as a better solution for each side of the equation.

Comment (farmerbb) [reply app/src/main/java/com/farmerbb/taskbar/ui/TaskbarController.java:460]: Ok, so an RRO will be used to theme `@drawable/tb_bliss` on Taskbar instead of `@drawable/bt_all_apps` on Boringdroid?  If so, is this good to merge?

Comment (electrikjesus) [reply app/src/main/java/com/farmerbb/taskbar/ui/TaskbarController.java:460]: Yes, I think the better solution is to theme each independently, as not all projects will use both solutions.

**Review by farmerbb — APPROVED** [review]
@utzcoz when this is ready, feel free to un-mark it as a draft and I'll merge.

**Comment by utzcoz** [comment]
@farmerbb I tested it on emulator with forcibly AG config, and it showed bliss logo correctly.

**Review by SaintPatricksCoin — APPROVED** [review]
